ttnn/cpp/ttnn/operations/pool/maxpool/max_pool.hpp

```diff
@@ -173,3 +179,53 @@ Tensor maxpool2d_new(const Tensor &input,
 
 }  // namespace tt_metal
 }  // namespace tt
+
+
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
this should be at the top in the file. Please double check if there are any copy paste errors